modules/local/arborview.nf

```diff
@@ -9,10 +9,8 @@ process ARBOR_VIEW {
     stageInMode 'copy' // Need to copy in arbor view html
 
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-    'docker.io/python:3.11.6' :
-    task.ext.override_configured_container_registry != false ? 
-    'docker.io/python:3.11.6' : 
-    'python:3.11.6' }"
+        'https://depot.galaxyproject.org/singularity/python%3A3.12' :
+        'biocontainers/python:3.12' }"
```

Comment:
I used the same container replacement for Aborview on SNVphyl, and it also seems to be working fine! I don't think there are any changes that will cause issues between python:3.12 and 3.11.6. We should do the same for gasclustering too, so that it is consistent.

Reply:
Sounds great. Thanks so much Steven :)